router-tests/block_operations_test.go

```diff
@@ -0,0 +1,202 @@
+package integration_test
+
+import (
+	"github.com/stretchr/testify/require"
+	"github.com/wundergraph/cosmo/router-tests/testenv"
+	"github.com/wundergraph/cosmo/router/core"
+	"github.com/wundergraph/cosmo/router/pkg/config"
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+)
+
+func TestBlockOperations(t *testing.T) {
+	t.Parallel()
+
+	t.Run("block mutations", func(t *testing.T) {
```

Comment:
Test naming could be improved because you are not only testing blocking behavior

```suggestion
	t.Run("test blocking rules for mutations", func(t *testing.T) {
```

Reply:
Improved